client/reader/following-edit/sort-controls.jsx

```diff
@@ -31,11 +31,11 @@ const FollowingEditSortControls = React.createClass( {
 		const sortOrder = this.props.sortOrder;
 
 		return (
-			<div className="following-edit-sort-controls">
+			<div className="following-edit__sort-controls">
 				<label htmlFor="sort-control-select">{ this.translate( 'Sort by' ) }</label>
 				<select id="sort-control-select" ref="sortControlSelect" className="is-compact" onChange={ this.handleSelectChange } value={ sortOrder }>
```

Comment:
Probably not necessary in this PR, but could we switch to using [select dropdown](http://calypso.dev:3000/devdocs/design/select-dropdown) instead of a select element here for consistency?